### PR TITLE
Working directory for Postman web

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: src/pages/docs/getting-started/settings.md [hunk 1]
```diff
@@ -71,6 +71,13 @@ However, delimiting the working directory can have some unintended security issu
 To learn more about this feature, refer to [Sending body data](/docs/sending-requests/requests/#sending-body-data).
 
 * **Location:** Path to local directory containing postman files. The default path populated as a placeholder is `~/Postman/files`.
+
+For Postman app, you can modify the working directory. While uploading a file to the app during a POST request, you will be able to view the file only when it is present in the working directory.
+
+<img alt="Working directory on web" src="https://assets.postman.com/postman-docs/working-directory-web.jpg" width="350px">
+
+For Postman web, the working directory cannot be modified. When you upload a file on the web, it creates a folder with a random alphanumeric name (e.g. 6OPkYiXVr) in the `~/Postman/files` directory whether or not the file is selected within the working directory or outside the working directory. Since the file’s relative path to the working directory remains unchanged, the new folder created would prevent you from sharing the requests with other members seamlessly. You can view all the files within the newly created folders in the working directory.
```
Comment (Raj-postman) [inline src/pages/docs/getting-started/settings.md:79]: > the new folder created would prevent you from sharing the requests with other members seamlessly.

The new folder created would enable you to continue testing with files selected anywhere from your Desktop but for team members to have a seamless experience - it is always suggested that the files remain within ~/Postman/files.

**Review by Raj-postman — APPROVED** [review]
Rest of it looks all good to me. Thank you!

File: src/pages/docs/getting-started/settings.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ However, delimiting the working directory can have some unintended security issu
 To learn more about this feature, refer to [Sending body data](/docs/sending-requests/requests/#sending-body-data).
 
 * **Location:** Path to local directory containing postman files. The default path populated as a placeholder is `~/Postman/files`.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:73]: postman should be Postman

File: src/pages/docs/getting-started/settings.md [hunk 3]
```diff
@@ -71,6 +71,14 @@ However, delimiting the working directory can have some unintended security issu
...
+
+For Postman app, you can modify the working directory. While uploading a file to the app during a POST request, you will be able to view the file only when it is present in the working directory.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:75]: "For Postman app" should be "For the Postman app"

File: src/pages/docs/getting-started/settings.md [hunk 4]
```diff
@@ -71,6 +71,14 @@ However, delimiting the working directory can have some unintended security issu
...
+
+<img alt="Working directory on web" src="https://assets.postman.com/postman-docs/working-directory-web.jpg" width="350px">
+
+For Postman web, the working directory cannot be modified. When you upload a file on the web, it creates a folder with a random alphanumeric name (e.g. 6OPkYiXVr) in the `~/Postman/files` directory whether or not the file is selected within the working directory or outside the working directory. Since the file’s relative path to the working directory remains unchanged,
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:79]: * Change "(e.g. 6OPkYiXVr)" to "(such as 6OPkYiXVr)" and put backticks around the name.
* MSG says not to use "whether or not" - can you reword that?

File: src/pages/docs/getting-started/settings.md [hunk 5]
```diff
@@ -71,6 +71,14 @@ However, delimiting the working directory can have some unintended security issu
...
+the new folder created would enable you to continue testing with files selected anywhere from your Desktop. To ensure a seamless experience for your team members, it is recommended that the files are located in `~/Postman/files` directory. You can view all the files within the newly created folders in the working directory.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:80]: * would enable you to / enables you to
* "recommended that the files are located in" - should that be "it's recommended that you keep files in the" or similar?

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
Couple of minor things...

File: src/pages/docs/getting-started/settings.md [hunk 6]
```diff
@@ -70,7 +70,14 @@ However, delimiting the working directory can have some unintended security issu
 
 To learn more about this feature, refer to [Sending body data](/docs/sending-requests/requests/#sending-body-data).
 
-* **Location:** Path to local directory containing postman files. The default path populated as a placeholder is `~/Postman/files`.
+* **Location:** Path to local directory containing Postman files. The default path populated as a placeholder is `~/Postman/files`.
```
Comment (saswatds) [inline src/pages/docs/getting-started/settings.md:73]: Is `Postman Files` a noun that people understand? 
To me, Location is the path in my local filesystem which Postman considers as the working directory. Users can put any files in there and Postman will consider them safe and persist the path relative to the working directory. Things outside the working directory are not considered safe.

File: src/pages/docs/getting-started/settings.md [hunk 7]
```diff
@@ -70,7 +70,14 @@ However, delimiting the working directory can have some unintended security issu
...
+
+For the Postman app, you can modify the working directory. While uploading a file to the app during a POST request, you will be able to view the file only when it is present in the working directory.
```
Comment (saswatds) [inline src/pages/docs/getting-started/settings.md:75]: Should we say postman desktop app? postman app could mean web app as well. Not sure what 
> you will be able to view the file only when it is present in the working directory
means. 

The users will be able to view all files, it is just that files outside the working directory, we do not persist the file path in the collection for security reasons. If you put your files in the working directory they will be persisted relative to the working directory.

File: src/pages/docs/getting-started/settings.md [hunk 8]
```diff
@@ -70,7 +70,14 @@ However, delimiting the working directory can have some unintended security issu
...
+
+<img alt="Working directory on web" src="https://assets.postman.com/postman-docs/working-directory-web.jpg" width="350px">
+
+For Postman web, the working directory cannot be modified. When you upload a file on the web, it creates a folder with a random alphanumeric name (such as `6OPkYiXVr`) in the `~/Postman/files` directory even when the file is selected within the working directory or outside the working directory. Since the file’s relative path to the working directory remains unchanged, the new folder created enables you to continue testing with files selected anywhere from your Desktop. To ensure a seamless experience for your team members, it is recommended that you keep files in the `~/Postman/files` directory. You can view all the files within the newly created folders in the working directory.
```
Comment (saswatds) [inline src/pages/docs/getting-started/settings.md:79]: The seamless experience for team members only applies to the postman desktop app and has a small caveat that the team members also maintain the same working directory structure.

**Review by saswatds — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**